wiki/FitNesseRoot/HsacExamples/SlimTests/UtilityFixtures/DateFormatFixture.wiki

```diff
@@ -45,18 +45,33 @@ This fixture is intended to be used as a library. (See below)
 |set date format           |MM/dd/yyyy                                                                                    |
 |check                     |format date        |11/14/2016         |as|dd-MM-yyyy           |14-11-2016                   |
 
-!4 Use as a library fixture from any othe fixture table
+!4 Use as a library fixture from any other fixture table
 
-The following will make the fixture available as alibrary with specified default values for format and time zone:
+When we use this fixture as a library we can call date format methods from any script table.
+
+By default a library will use system default date format and timezone.
+
+|library            |
+|date format fixture|
+
+Example usage of date formatting methods from a string fixture script table:
+
+|script |string fixture                                  |
+|$stamp=|timestamp for date|${date}                      |
```

Comment:
If system defaults are used, we need to specify the input format if we want to ensure passing travis builds